### PR TITLE
RGBD test added

### PR DESCRIPTION
**Pull request by prajurock**
# Motivation
RGBD toolkit Unit test wrriten

### CODE REVIEW

**Comment by prajurock** [comment]
@markus-hinsche  test should fail because #160  is not merged yet

File: src/common/rgbd_toolkit/test/test_rgbd.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+def test_get_files():
+    norm_rgb_time = np.asarray([668549.91427676, 668601.94653876, 668578.70910376, 668549.98114676, 668602.31434576])
+    rgb_path = [REPO_DIR + '/qr_code/qr_code_test/rgb/rgb_1584994919-58mqazoorz_1591550095148_100_668549.914276759.jpg', REPO_DIR + '/qr_code/qr_code_test/rgb/rgb_1584994919-58mqazoorz_1591550095148_101_668601.946538759.jpg',
+                REPO_DIR + '/qr_code/qr_code_test/rgb/rgb_1584994919-58mqazoorz_1591550095148_102_668578.709103759.jpg', REPO_DIR + '/qr_code/qr_code_test/rgb/rgb_1584994919-58mqazoorz_1591550095148_100_668549.981146759.jpg', REPO_DIR + '/qr_code/qr_code_test/rgb/rgb_1584994919-58mqazoorz_1591550095148_101_668602.314345759.jpg']
+    norm_pcd_time = np.asarray([668578.83466482, 668550.01378682, 668601.96653282])
+    pcd_path = [REPO_DIR + '/qr_code/qr_code_test/pc/pc_1584994919-58mqazoorz_1591550095148_102_000.pcd',
+                REPO_DIR + '/qr_code/qr_code_test/pc/pc_1584994919-58mqazoorz_1591550095148_100_000.pcd', REPO_DIR + '/qr_code/qr_code_test/pc/pc_1584994919-58mqazoorz_1591550095148_101_000.pcd']
+    files = get_files(norm_rgb_time, rgb_path, norm_pcd_time, pcd_path)
+    print(REPO_DIR)
```
Comment (markus-hinsche) [inline src/common/rgbd_toolkit/test/test_rgbd.py:44]:
```suggestion
```

**Comment by markus-hinsche** [comment]
image

The PCD files are very big (1.5MB each). They blow up the git repo unnecessarily. Can we make them smaller (few KB ideally).

**Comment by markus-hinsche** [comment]
Tests are failing

**Review by markus-hinsche — CHANGES_REQUESTED** [review]
The checks are still failing. Please request a review once the checks are passing

**Pull request merged**